docs/changes/11982.bugfix.rst

```diff
@@ -0,0 +1 @@
+Now accepting UCDs containing phot.color (Fixes: #11981).
```

Comment:
Please move this file to `docs/changes/io.votable` folder and squash out the commits into one commit. Thanks!

Reply:
@saimn or @Cadair , how do we fix the change log check to catch cases like this?

Reply:
Your means for the directory ? The pre-commit should already take care of this.

Reply:
I guess we'll just have to get #11945 merged at some point... 🤞